fxdc/config.py

```
@@ -126,14 +249,73 @@ def __init__(self, data):
+                    if not field.blank:
+                        data["notblank"].append(name)
+                    if field.desc:
+                        data["description"][name] = field.desc
+            return data
+
         def wrapper(class_: T) -> T:
             if self.get_class_name(class_) in self.custom_classes_names:
                 raise ClassAlreadyInitialized(
-                    f"Class {name} already exists"
+                    f"Class {self.get_class_name(class_)} already exists"
                 )
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)